sympy/solvers/ode.py

```diff
@@ -1218,8 +1218,7 @@ class in it.  Note that a hint may do this anyway if
                 matching_hints[s + "_Integral"] = r
                 if undetcoeff['test']:
                     r['trialset'] = undetcoeff['trialset']
-                    matching_hints["nth_linear_constant_coeff_undetermined_"
-                        "coefficients"] = r
+                    matching_hints["nth_linear_constant_coeff_undetermined_coefficients"] = r
```

Comment:
The wrap was probably introduced to stay below the 80-column limit.
I suggest moving the string to a line of its own.